src/go/pt-galera-log-explainer/display/timelinecli_test.go

```diff
@@ -0,0 +1,186 @@
+package display
+
+import (
+	"testing"
+
+	"github.com/percona/percona-toolkit/src/go/pt-galera-log-explainer/types"
+	"github.com/percona/percona-toolkit/src/go/pt-galera-log-explainer/utils"
+)
+
+func TestTransitionSeparator(t *testing.T) {
+	tests := []struct {
```

Comment:
It seems it's safe to run this test in parallel:
```suggestion
func TestTransitionSeparator(t *testing.T) {
        t.Parallel()
	tests := []struct {
```

Reply:
Agreed, committed